<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolboxWidget</class>
 <widget class="QWidget" name="ToolboxWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,10,1">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Tool Box Pannel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_filename">
     <property name="text">
      <string>File Name : </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_Explore">
         <attribute name="title">
          <string>Explore</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="1">
             <widget class="QSpinBox" name="X_spinbox"/>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="T_spinbox"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Y</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSlider" name="Y_slider">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>T</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="Z_spinbox"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Z</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QSlider" name="T_slider">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="Y_spinbox"/>
            </item>
            <item row="0" column="2">
             <widget class="QSlider" name="X_slider">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QSlider" name="Z_slider">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>X</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1" colspan="2">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Viewing Options</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Select desired View :</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radio_X">
                 <property name="text">
                  <string>X</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radio_Y">
                 <property name="text">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radio_Z">
                 <property name="text">
                  <string>Z</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radio_all">
                 <property name="text">
                  <string>All (this needs multiple EPICS Viewers)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextBrowser" name="textBrowser">
                 <property name="html">
                  <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:6.6pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.25pt; font-weight:600;&quot;&gt;Multiple View: &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.25pt;&quot;&gt;to set up multiple slice view, please open more EPICS Viewers. Set up the channel names in &lt;/span&gt;&lt;span style=&quot; font-size:8.25pt; font-style:italic; text-decoration: underline;&quot;&gt;setting&lt;/span&gt;&lt;span style=&quot; font-size:8.25pt;&quot;&gt; dialogue. &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.25pt; font-weight:600;&quot;&gt;Note&lt;/span&gt;&lt;span style=&quot; font-size:8.25pt;&quot;&gt;: It is recommended to use Y,Z and All View options, only when the volume is loaded in the RAM. Large HDF files tend to be slow when slicing.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_rotate">
         <attribute name="title">
          <string>Rotate</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3"/>
        </widget>
        <widget class="QWidget" name="tab_crop">
         <attribute name="title">
          <string>Crop</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <widget class="QPushButton" name="btn_apply">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cancel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_load_setting">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Settings</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radio_X</sender>
   <signal>toggled(bool)</signal>
   <receiver>X_slider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>X_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>X_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>X_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>X_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Y_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Y_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Y_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Y_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Z_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Z_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Z_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Z_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>T_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>T_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>T_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>T_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_Y</sender>
   <signal>toggled(bool)</signal>
   <receiver>Y_slider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_Z</sender>
   <signal>toggled(bool)</signal>
   <receiver>Z_slider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_all</sender>
   <signal>toggled(bool)</signal>
   <receiver>Z_slider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_all</sender>
   <signal>toggled(bool)</signal>
   <receiver>Y_slider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_all</sender>
   <signal>toggled(bool)</signal>
   <receiver>X_slider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
